skimage/segmentation/manual_segmentation.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.patches import Polygon
+from matplotlib.collections import PatchCollection
+
+
+def manual(image, speed_up=1):
+    """Return a binary image based on the selections made with mouse clicks.
+
+    Parameters
+    ----------
+    image : (M, 2) array
+        Grayscale or RGB 2D image.
+
+    speed_up : int
+        Skips vertices in integer steps to speed up mask generation. This
+        must be a non-zero integer.
+
+    Returns
+    -------
+    mask : (M, 2) binary array
+        Segmented binary mask.
+
+    Notes
+    -----
+    Use the cursor to draw objects. Increasing speed_up value will 
+    generate the mask faster, but the objects will be less smoother. 
+
+    Examples
+    --------
+    >>> from skimage import data, segmentation
+    >>> camera = data.camera()
+    >>> mask = segmentation.manual(camera)
+    >>> # Use the cursor to draw objects
+    >>> io.imshow(mask)
+    >>> io.show()
+
+    """
+    image = np.squeeze(image)
+
+    if image.ndim not in (2, 3):
+        raise TypeError('Only 2-D images or 3-D images supported.')
+
+    manual.list_of_verts = []
+    manual.polygons_selection = []
+    manual.polygon_list = []
+
+    _select_lasso(image)
```

Comment:
In general we prefer a functional programming style in scikit-image. So, similarly, in this case, I would go with `verts = _select_lasso(image)`.